src/components/Homepage/services.js

```diff
@@ -107,13 +107,21 @@ const Services = ({ services }) => (
 
               {service.pageReady ? (
                 <Padding bottom={1.5}>
-                  <StyledLink to={`/${service.slug}`}>Learn more</StyledLink>
+                  <StyledLink
+                    to={`/${service.slug}`}
+                    title={`Learn more - ${service.title}`}
```

Comment:
would "Learn more about our ${service.title} service" be more descriptive / better?

Reply:
updated